.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Build
+
+on:
+  push:
+
+env:
+  minikube_version: v1.15.1 # Latest as of 2020-11-19
+  E2E_START_MINIKUBE: yes_please
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: make
+
+  e2e-test-1-16:
+    name: E2E Tests for 1.16
+    runs-on: ubuntu-18.04
+    needs: [ build ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: manusa/actions-setup-minikube@v2.0.1
+        with:
+          minikube version: ${{ env.minikube_version }}
+          kubernetes version: v1.16.15
```

Comment:
instead of copying the same code you can use a matrix and running everything in parallel

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
Absolutely, the Build.yml is a heavy WIP for now :)